templates/security/reset_password/_layout.html.twig

```diff
@@ -0,0 +1,9 @@
+{% extends '@Tabler/security.html.twig' %}
+
+{% block login_social_auth %}{% endblock %}
+
+{% block login_box_error %}
+    {% for flashError in app.flashes('reset_password_error') %}
+        <div class="alert alert-important alert-danger">{{ flashError }}</div>
```

Comment:
We should use the alert macro, even if that means more code.

Reply:
True 👍